public/docs/ts/latest/quickstart.jade

```diff
@@ -10,479 +10,471 @@ include ../../../_includes/_util-fns
+  1. Angular's browser `bootstrap` function
+  1. The application root component that we just wrote.
+  
+  We import both. Then we call `bootstrap`, passing in the **root component type**, 
+  `AppComponent`.
+  
+.l-sub-section
+  :marked
+    ### bootstrapping is platform-specific
+    Note that `bootstrap` function comes from a *different library*, `angular2/platform/browser`.
+    Most Angular applications run only in a browser and we'll call the bootstrap function from
+    this library most of the time. 
+    
+    It is possible to load a component in a different enviroment such as we might do
+    if we wished to render the first page of our application on the server.
+    That would require a different kind of bootstrap function. 
+:marked
+  We've asked Angular to launch the app in a browser with our component at the root.
+  Where will Angular put it? 
 
 .l-main-section
-  :marked
-    ## Install npm packages locally
-
-    We'll replace the web-based scripts in our `index.html` with
-    scripts resident on our local machine.
-    We get those scripts by installing two runtime `npm` packages into our project.
-
-    >**angular2** - the Angular 2 library.
-
-    >**systemjs** - an open-source library that provides module loading.
-
-    We'll also install two development tools:
-
-    >**typescript** - the TypeScript compiler
-
-    >**[live-server](https://www.npmjs.com/package/live-server "Live-server")** - the static file server that reloads the browser when files change.
-    We may have loaded it earlier. We're doing it again
-    locally in our project so we are no longer vulnerable to
-    a global uninstall or version change.
-
-    **Open** a terminal window at our application's **root folder**
-
-    Enter these commands:
-    ```
-    npm init -y
-    npm i angular2@2.0.0-alpha.44 systemjs@0.19.2 --save --save-exact
-    npm i typescript live-server --save-dev
-    ```
-
-    These commands both *install* the packages and *create* an npm `package.json` that will
-    help us develop and maintain our application in the future.
-    The essence of our `package.json` should look like this:
+:marked
+  ## Add the `index.html`
+  
+  Angular displays our application in a specific location on our `index.html`.
+  It's time to create that file.
+
+  We won't put our `index.html` in the `app/` folder. 
+  We'll locate it **up one level, in the project root folder**.
+  
+code-example(format="").
+  cd ..
+:marked
+  Now create the`index.html` file and paste the following lines:
++makeExample('quickstart/ts/index.html', null, 'index.html')
+:marked
+  There are three noteworthy sections of HTML:
 
-  +makeJson('quickstart/ts/package.json', { paths: 'name, version, dependencies, devDependencies'})
+  1. We load the JavaScript libraries we need.<br/>
 
-  :marked
-    There is also a `scripts` section. **Find and replace** it with the following:
+  2. We configure something called `System` and ask it to import the
```

Comment:
Perhaps it's worth it to mention why locally we configure it to read `.js` files, but in the plunker we go for `.ts` files? Something about tooling and code completion.